neurom/fst/__init__.py

```diff
@@ -1,177 +1,7 @@
-# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
```

Comment:
We should still retain this license shouldn't we?

Reply:
From what I understood, having the licence at the root of your project is enough to protect it.

Reply:
no, it has to be in every file.

Reply:
Is it a BlueBrain directive ? Because most of the resources I've found on the internet seem to indicate the opposite. 
https://stackoverflow.com/questions/845895/putting-license-in-each-code-file
https://github.com/NethermindEth/nethermind/issues/1067

Reply:
Nevermind, @haleepfl showed me our internal directives where it is clearly specified.

Reply:
Well, well, our internal guidelines are not very clear about BSD. I have created a ticket about this discussion here: https://github.com/BlueBrain/NeuroM/issues/799